bom/models.py

```diff
@@ -90,7 +90,7 @@ class Meta():
         unique_together = ['number_class', 'number_item', 'number_variation', 'organization', ]
 
     def full_part_number(self):
-        return "{0}-{1}-{2}".format(self.number_class.code,
+         return "{0}-{1}-{2}".format(self.number_class.code,
```

Comment:
@lcesero can you remove this extra space?

Reply:
Space has be removed.